packages/liveblocks-node/src/client.ts

```diff
@@ -157,11 +257,11 @@ export class Liveblocks {
       | string // Shorthand for userId
       | Identity,
     options?: {
-      userInfo: unknown;
+      userInfo: Json;
```

Comment:
This should also be `IUserInfo` type (the backend will throw if you'd pass it anything else).